packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -211,7 +205,7 @@ export const useChatMyMessageStyles = makeStyles({
     top: '-19px',
     right: '0',
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
-    zIndex: 1,
+    zIndex: 2,
```

Comment:
wh ydo we need to update this value? Could you add a comment in the code too?

Reply:
the outline will show over top of the more actions button, as they are seen as one component and it's outside the bounds

Reply:
So we have to make sure the more actions is high enough in z order

Reply:
I can add a comment

Reply:
In this case the focus border is now being handled by fluent, which does currently account for high contrast themes.